### PR TITLE
Documentation/(sakuli/sakuli dashboard#61)/dashboard info

### PR DESCRIPTION
**Pull request by cliffle**

### CODE REVIEW

File: data/en/features_enterprise.yml [hunk 1]
```diff
@@ -74,6 +74,14 @@ featuresItem:
...
-            
\ No newline at end of file
+
+      - icon : fas fa-desktop
+        title : Sakuli-Dashboard
+        content:
+            - paragraph : With the Sakuli dashboard, you can track the monitoring metrics of your application alongside a live video feed of your Sakuli check.
+            - heading: Features
+              list:
+                  - liEl: Configurable play button to perform actions on Kubernetes/Openshift clusters
```
Comment (ZNicoZ) [inline data/en/features_enterprise.yml:84]:
```suggestion
                  - liEl: Configurable play button to start sakuli tests on Kubernetes/Openshift clusters
```

File: data/en/features_enterprise.yml [hunk 2]
```diff
@@ -74,6 +74,14 @@ featuresItem:
...
+                  - liEl: Trigger your Sakuli test on your cluster with the play button and watch it perform in real-time
```
Comment (ZNicoZ) [inline data/en/features_enterprise.yml:85]:
```suggestion
                  - liEl: Watch your Sakuli test in real-time with the integrated VNC display
```

File: data/en/features_enterprise.yml [hunk 3]
```diff
@@ -74,6 +74,14 @@ featuresItem:
...
+                  - liEl: Start your Sakuli test automatically with a cronjob to monitor your application continuously
```
Comment (ZNicoZ) [inline data/en/features_enterprise.yml:86]:
```suggestion
                  - liEl: Automated start of your Sakuli tests in defined cycles to monitor your application continuously
```

File: data/en/features_enterprise.yml [hunk 4]
```diff
@@ -74,6 +74,14 @@ featuresItem:
...
+                  - liEl: Embed your monitoring metrics on the dashboard to check the performance of your application
```
Comment (ZNicoZ) [inline data/en/features_enterprise.yml:87]:
```suggestion
                  - liEl: Embed your monitoring frontend to the dashboard and monitor the performance of your application
```

File: data/de/features_enterprise.yml [hunk 5]
```diff
@@ -75,5 +75,15 @@ featuresItem:
...
\ No newline at end of file
+      - icon : fas fa-desktop
+        title : Sakuli-Dashboard
+        content:
+            - paragraph : Start und Überwachung Ihres Sakuli Test und Darstellung der Monitoring-Metriken Ihrer Anwendung in real-time.
+            - heading: Features
+              list:
+                  - liEl: Konfigurierbarer Start Button um Sakuli Tests auf einem Kubernetes/Openshift Cluster zu starten
+                  - liEl: Beobachten Sie die Testausführung live über das im Dashboard eingebundene VNC Display
+                  - liEl: Automatisierter Start Ihrer Sakuli Tests in bestimmten Zeitintervallen, um Ihre Anwendung fortlaufend zu überwachen
```
Comment (svettwer) [inline data/de/features_enterprise.yml:86]:
```suggestion
                  - liEl: Konfigurierbarer Start Button um einen Sakuli Test auf einem Kubernetes/Openshift Cluster auszuführen.
                  - liEl: Beobachten Sie die Testausführung live über das im Dashboard eingebundene VNC Display.
                  - liEl: Automatisierter Start Ihrer Sakuli Tests in bestimmten Zeitintervallen, um Ihre Anwendung fortlaufend zu überwachen.
```

File: data/de/features_enterprise.yml [hunk 6]
```diff
@@ -75,5 +75,15 @@ featuresItem:
...
+                  - liEl: Einbindung Ihres Monitoring Frontends um die Leistung Ihrer Anwendung kontinuierlich zu überwachen.
```
Comment (svettwer) [inline data/de/features_enterprise.yml:87]:
```suggestion
                  - liEl: Einbindung Ihres Monitoring Frontends um die Leistung Ihrer Anwendung kontinuierlich im Blick zu behalten.
```

File: data/en/features_enterprise.yml [hunk 7]
```diff
@@ -74,6 +74,16 @@ featuresItem:
...
             - paragraph : The Sakuli DEV team tracks security incidents of underlying software components regularly and updates Sakuli components with the latest security fix releases shortly after appearance.
-      
-                  
-            
\ No newline at end of file
+
+      - icon : fas fa-desktop
+        title : Sakuli-Dashboard
+        content:
+            - paragraph : Start and supervision of your Sakuli test und display of monitoring data in real-time.
```
Comment (svettwer) [inline data/en/features_enterprise.yml:81]:
```suggestion
            - paragraph : Start and supervision of your Sakuli test and display of monitoring data in real-time.
```

File: data/en/features_enterprise.yml [hunk 8]
```diff
@@ -74,6 +74,16 @@ featuresItem:
...
+            - heading: Features
+              list:
+                  - liEl: Configurable play button to start sakuli tests on Kubernetes/Openshift clusters
+                  - liEl: Watch your Sakuli test in real-time with the integrated VNC display
+                  - liEl: Automated start of your Sakuli tests in defined cycles to monitor your application continuously
+                  - liEl: Embed your monitoring frontend to the dashboard and monitor the performance of your application
```
Comment (svettwer) [inline data/en/features_enterprise.yml:87]:
```suggestion
                  - liEl: Configurable play button to start Sakuli tests on Kubernetes/Openshift clusters.
                  - liEl: Watch your Sakuli test in real-time with the integrated VNC display.
                  - liEl: Automated scheduling of your Sakuli test to monitor your application continuously.
                  - liEl: Embed your monitoring frontend to the dashboard and monitor the performance of your application.
```

**Review by svettwer — CHANGES_REQUESTED** [review]
Sorry for another iteration but as nico is on vacation, I took his review task.

**Review by svettwer — APPROVED** [review]

**Pull request merged**